app/controllers/notifications_controller.rb

```diff
@@ -1,4 +1,6 @@
 class NotificationsController < ApplicationController
   def index
+    notifications = current_user.notifications.unread.order(created_at: :desc)
```

Comment:
use `includes` to lazy load any info necessary to display on the notification (i.e speaker or session info)